### DISCUSSION TITLE
howto rebase/merge with different origin iteration

### DISCUSSION THREAD

**Discussion by magana1** [swellstores/community#17] [Help] [answered]
sorry if the question seems stupid to you.  I made some changes to the origin theme( new section design). how to merge my new sections with the latest origin theme version?

**Comment by greph** [comment, marked as answer]
Hey @magana1 , are you looking to make a contribution to the Swell template or simply push up your changes back to your Swell-hosted storefront? If the latter, you can simply make a local commit and push your changes back up to remote. 

If you'd like to also pull in the latest changes from the Origin template, you can add a second remote repo linking to the [public Origin repo ](https://github.com/swellstores/origin-theme) and then merge in those newer changes. Finally, I'd recommend adding the --force flag when pushing changes back to your Swell-hosted remote repo.

I hope this helps - let me know if you have any questions!
